lesson11/three/three.go

```diff
@@ -0,0 +1,11 @@
+package three
+
+import "io"
+
+func printAll(w io.Writer, values ...interface{}) {
```

Comment:
why "printAll" since I expect to print strings only? better "printStrings" or smth like that